pyproject.toml

```diff
@@ -45,6 +46,7 @@ repository = 'https://github.com/mbed-ce/cy_serial_bridge/'
 
 [tool.poetry.scripts]
 cy_serial_cli = 'cy_serial_bridge.cli:main'
+cy_flash = 'cy_serial_bridge.flash:main'
```

Comment:
I think this is pointing to a script that doesn't exist (in this PR at least?)

Reply:
Ahh sorry - it's a utility for flashing NOR flash chips and similar devices that I'm working on. You need something a bit more complicated than the SPI write of the general CLI, and there are many options, so I thought a separate cy_flash cli program is the way to go.
Its a work in progress, the branch for it is here: https://github.com/runger1101001/cy_serial_bridge/tree/cy_flash_utility